cmd/query/main_test.go

```diff
@@ -108,6 +108,26 @@ func TestSliceErrorHTTPResponse(t *testing.T) {
 			},
 			testSliceRequest{},
 		},
+		sliceTest{
+			baseTest{
+				name:           "Invalid json GET request",
+				method:         http.MethodGet,
+				jsonRequest:    "nil",
```

Comment:
Maybe instead of nil identification use something like "MODIFIER: skip query parameter" so that whoever reads the tests has an idea that something special is about to happen in case of this value.

I am not 100% happy with it, but for now can't think of anything better. Your suggestion seems to be the best.